src/System.IO/src/System.IO.csproj

```diff
@@ -16,16 +16,19 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50aot_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50aot_Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard13aot_Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard13aot_Release|AnyCPU'" />
 
   <PropertyGroup Condition="'$(TargetGroup)' == 'net46'">
     <ResourcesSourceOutputDirectory>None</ResourcesSourceOutputDirectory>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetGroup)' == 'netcore50aot'">
+  <PropertyGroup Condition="'$(TargetGroup)' == 'netcore50aot'  or '$(TargetGroup)' == 'netstandard13aot'">
     <SkipCommonResourcesIncludes>true</SkipCommonResourcesIncludes>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ExcludeAssemblyInfoPartialFile>true</ExcludeAssemblyInfoPartialFile>
     <StringResourcesPath>Resources/Strings.netcore50aot.resx</StringResourcesPath>
+    <SkipValidatePackageTargetFramework Condition="'$(TargetGroup)' == 'netstandard13aot'">true</SkipValidatePackageTargetFramework>
```

Comment:
If you just needed to exclude the System.Private references you can use ValidateIgnoreReference to do so.